notebooks/patch_to_point/patch_to_point.ipynb

```diff
@@ -0,0 +1,689 @@
+    "    temporal_context = TemporalContext(row.start_date, row.end_date)  # In the end we'll probably don't need this. Leaving this in, just in case for now.\n",
+    "\n",
+    "    backend = Backend.TERRASCOPE\n",
+    "    backend_context = BackendContext(backend)\n",
+    "\n",
+    "    fetch_type = FetchType.TILE\n",
+    "\n",
+    "    # The S1 and S2 cubes are loaded from their respective STAC collections\n",
+    "    s1_cube = connection.load_stac(row.s1_collection_path, temporal_extent=[row.start_date, row.end_date])\n",
+    "    s1_cube = median_compositing(s1_cube, period=\"month\")\n",
+    "    \n",
+    "    s2_bands = [\n",
+    "            \"S2-L2A-B01\",\n",
+    "            \"S2-L2A-B02\",\n",
+    "            \"S2-L2A-B03\",\n",
+    "            \"S2-L2A-B04\",\n",
+    "            \"S2-L2A-B05\",\n",
+    "            \"S2-L2A-B06\",\n",
+    "            \"S2-L2A-B07\",\n",
+    "            \"S2-L2A-B08\",\n",
+    "            \"S2-L2A-B8A\",\n",
+    "            \"S2-L2A-B11\",\n",
+    "            \"S2-L2A-B12\"\n",
+    "        ]  # Not extracting SCL and distance to cloud, can easily be added if needed\n",
+    "    s2_cube = connection.load_stac(row.s2_collection_path, temporal_extent=[row.start_date, row.end_date])\n",
+    "    s2_cube = s2_cube.filter_bands(s2_bands)\n",
+    "    s2_cube = median_compositing(s2_cube, period=\"month\")\n",
+    "\n",
+    "    cube = s1_cube.merge_cubes(s2_cube)\n",
+    "\n",
+    "    # For DEM we can use worldcereal_classification's raw_datacube_DEM\n",
+    "    dem_cube = raw_datacube_DEM(\n",
+    "        connection=connection,\n",
+    "        backend_context=backend_context,\n",
+    "        spatial_extent=spatial_extent,\n",
+    "        fetch_type=fetch_type,\n",
+    "        )\n",
+    "    dem_cube = dem_cube.resample_cube_spatial(s2_cube, method=\"bilinear\")\n",
+    "    dem_cube = dem_cube.linear_scale_range(0, 65534, 0, 65534)  # Will not work for geoparquet output (openeo-devs are aware, but are not giving it priority)\n",
+    "\n",
+    "    # METEO is loaded from a pre-composited STAC collection\n",
+    "    meteo_cube = connection.load_stac('/data/users/Public/vincent.verelst/world_cereal/agera5_monthly/stac/collection.json', spatial_extent=dict(spatial_extent), temporal_extent=[row.start_date, row.end_date])\n",
+    "    meteo_cube.result_node().update_arguments(featureflags={\"tilesize\": 1})\n",
+    "    meteo_cube = meteo_cube.rename_labels(\n",
+    "            dimension=\"bands\", target=[\"AGERA5-PRECIP\", \"AGERA5-TMEAN\"]\n",
+    "        )\n",
+    "\n",
+    "    cube = s1_cube.merge_cubes(s2_cube)\n",
+    "    cube = cube.merge_cubes(dem_cube)\n",
+    "    cube = cube.merge_cubes(meteo_cube)\n",
+    "    \n",
+    "    # This is just a placeholder for the RDM API interaction function. In this example we just load in some random points from parquet as an example.\n",
```

Comment:
these random points should be the sampling procedure in the ground truth I guess? What will happen if a point somehow falls outside the available patches?

Reply:
The patches are constructed as: reference geometry (from public RDM) -> take centroid -> buffer to square of 64x64
The points are constructed as: same reference geometry (from public RDM) -> take centroid
So by construction the points wshould always fall within the patches. 

For custom user reference data, the points may fall outside of the patches, but it was my understanding that we would not use the cached patch data for these extractions.

Reply:
ah yeah ok but we'll add another point sampling method afterwards to not always have to stick (exclusively) to centroids. We just need to make sure we then don't make mistakes and fall outside the patches.

Reply:
This sampling will be an important thing, to be aligned with @cbutsko how we can do this correctly.

Reply:
From discussion with @cbutsko we should find a way to flag the original center polygon('s centroid) as the initially targeted sample, and then add the "collateral" fields('s centroids) from the patch with another flag.